front/app/containers/Admin/projects/components/PDFExportModal/messages.ts

```diff
@@ -38,9 +38,9 @@ export default defineMessages({
     defaultMessage:
       'Survey logic will not be reflected in the downloaded PDF. Paper respondents will see all survey questions.',
   },
-  personalDataExplanation: {
-    id: 'app.containers.Admin.projects.all.personalDataExplanation2',
+  personalDataExplanation4: {
+    id: 'app.containers.Admin.projects.all.personalDataExplanation4',
     defaultMessage:
-      'Check the box below, if you want your downloaded PDF to contain name, last name, and email fields. Upon upload of the paper form, we will use that user data, to auto-generate an account for the offline survey respondent.',
+      'This option will add the first name, last name, and email fields to the exported PDF. Upon uploading the paper form, we will use that data to auto-generate an account for the offline survey respondent. Alternatively, you can use it to manually create an account for the offline survey respondent.',
```

Comment:
I think the only way to manually create accounts (outside of the idea import) is to create invites. I would either change the text to 'Alternatively, you can use it to manually create an invite to the platform for the offline survey respondent.' or just remove the text entirely.

Reply:
Yeah, removing it entirely makes more sense!